src/login/logind-button.c

```diff
@@ -48,6 +66,8 @@ Button* button_new(Manager *m, const char *name) {
                 return mfree(b);
         }
 
+        b->mods_depressed = Button_Modifier_None;
```

Comment:
strictly speaking, once the device is opened you'd have to initialize this via EVIOCGKEY

Reply:
I am hoping I am using the ioctl correctly